docs/logical_data_model.puml

```diff
@@ -443,6 +443,7 @@ class Goals{
  source : enum
  status : varchar(255)
  timeframe : text
+ * prestandard : boolean : false
```

Comment:
Was there any thought give to using a "version" or "generation" column instead? Just wondering about the _next_ iteration of the goals experiment - would we then add another "pre-**X**" column or would it make sense to increment a version number instead?

Reply:
We chatted about it at eng sync and decided that for now we'll leave it as-is, but if we end up with another version of Goals then we might turn prestandard into a virtual column or something.